tests/modules/programs/mpv/mpv-stubs.nix

```
@@ -1,26 +1,27 @@
-{ pkgs, ... }:
-
 {
   nixpkgs.overlays = [
-    (self: super: {
-      mpvScript = pkgs.runCommandLocal "mpvScript" { scriptName = "something"; }
+    (final: prev: {
+      mpvScript = prev.runCommandLocal "mpvScript" { scriptName = "something"; }
         "mkdir $out";
 
-      mpv-unwrapped = super.mpv-unwrapped.overrideAttrs {
-        builder = pkgs.writeShellScript "dummy" ''
-          PATH=${pkgs.coreutils}/bin
+      mpv-unwrapped = let
+        lua = prev.emptyDirectory.overrideAttrs {
+          luaversion = "0";
+          passthru.withPackages = pkgsFn: prev.emptyDirectory;
+        };
+        mpv-unwrapped' = prev.mpv-unwrapped.override { inherit lua; };
+      in mpv-unwrapped'.overrideAttrs {
+        buildInputs = [ ];
+        nativeBuildInputs = [ ];
+        builder = prev.writeShellScript "dummy" ''
+          PATH=${prev.coreutils}/bin
```

I think you'll want to refer to the final coreutils, otherwise it'll refer to some intermediate version that causes a rebuild.
```suggestion
          PATH=${final.coreutils}/bin
```